graphql_api/tests/test_test_result_legacy.py

```diff
@@ -0,0 +1,218 @@
+# TODO - remove this file with #2291
```

Comment:
This file is a straight copy of `test_test_result.py` prior to adjusting the queries to handle the extra level of nest